.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,3 @@
+Please make sure that these boxes are checked before closing your pull request.
+- [ ] Set milestone for this PR
+- [ ] Add a brief explanation about this PR in CHANGELOG.md
```

Comment:
I would like to add another item something like "Added unit test for this change".